app/models/user.rb

```diff
@@ -16,4 +16,10 @@ def queued_video?(video)
   def owner?(queue_item)
     queue_items.include?(queue_item)
   end
+
+  def normalize_queue_item_position
+    queue_items.each_with_index do |queue_item, index|
+      queue_item.update_attributes(position: (index + 1))
```

Comment:
Use `update` instead of `update_attributes` as its shorter and does the same thing.